doc/admin-guide/files/records.config.en.rst

```diff
@@ -1674,8 +1693,7 @@ Proxy User Variables
    -  ``Accept-Encoding: gzip`` (if the header has ``gzip`` or ``x-gzip`` with any ``q``) **OR**
    -  *blank* (for any header that does not include ``gzip``)
 
-   This is useful for minimizing cached alternates of documents (e.g. ``gzip, deflate`` vs. ``deflate, gzip``). Enabling this option is
-   recommended if your origin servers use no encodings other than ``gzip``.
+   This is depricated.  Use **proxy.config.http.normalize_ae** instead.
```

Comment:
I think there's an explicit `:deprecated:` similar to `:reloadable:` which should be used as well.